sdk/core/Azure.Core/src/Diagnostics/AzureCoreEventSource.cs

```diff
@@ -37,6 +37,12 @@ internal sealed class AzureCoreEventSource : AzureEventSource
 
         public static AzureCoreEventSource Singleton { get; } = new AzureCoreEventSource();
 
+        [NonEvent]
+        public bool IsEnabledInformational() => IsEnabled(EventLevel.Informational, EventKeywords.None);
```

Comment:
nit: if we find these methods useful in core, they can be useful in other sources and could be moved to the base `AzureEventSource`

Reply:
Moved `WriteEvent` helpers to the `AzureEventSource`